packages/wrangler/src/constellation/listRuntime.tsx

```diff
@@ -0,0 +1,65 @@
+import Table from "ink-table";
```

Comment:
There's a lot of duplicated logic across the `list*` files. Can it be extracted into common helper functions?

Reply:
Done.